Tasks/Xcode/xcode.ts

```diff
@@ -363,6 +362,11 @@ async function run() {
                     }
 
                     if (xcodeVersion >= 9 && exportOptions === 'auto') {
+                        const cloudEntitlement = await sign.getCloudEntitlement(embeddedProvProfiles[0], exportMethod);                        
```

Comment:
Is this relevant to Xcode 8 as well? If so, add a separate if block.

Reply:
Nope this is only relevant from Xcode 9 and after.